src/__tests__/networks.integration.test.ts

```diff
@@ -9,14 +9,45 @@ describe('ethrResolver (alt-chains)', () => {
   const { address } = interpretIdentifier(addr)
   const checksumAddr = address
 
-  describe('eth-testnets', () => {
-    it('resolves on ropsten when configured', () => {
+  describe('eth-networks', () => {
+    it('resolves on mainnet with versionId', async () => {
+      const resolver = new Resolver(getResolver({ infuraProjectId: '6b734e0b04454df8a6ce234023c04f26' }))
+      const result = await resolver.resolve('did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388?versionId=12090174')
+      expect(result).toEqual({
+        didDocumentMetadata: {
+          nextVersionId: '12090175',
+          nextUpdate: '2021-03-22T18:14:29.000Z',
```

Comment:
milliseconds should be dropped --> 2021-03-22T18:14:29Z

Reply:
correct.
but that is going to be part of a different PR